widgets/price-table/js/pricetable.js

```diff
@@ -1,4 +1,28 @@
 jQuery( function($){
+    $('.ow-pt-columns-atom.sow-equalize-row-heights').each(function () {
```

Comment:
So right now, we only have one visual style for the price table (atom), but in the future, we'll probably be adding more. Is there any way to make this style independent? Maybe target the generic widget wrapper, which should be one level up from `.sow-equalize-row-heights`.